### ISSUE TITLE
AdaptiveCardHost: FluentUITextInput - cannot read properties of undefined (reading 'value')

### ISSUE THREAD

**Issue by phillipharding** [pnp/sp-dev-fx-controls-react#1591]
#### Category
[ ] Enhancement

[X] Bug

[ ] Question

#### Version

Please specify what version of the library you are using: [3.12.0]

#### Expected / Desired Behavior / Question
Using an **Input.Text** control in a simple adaptive card should work as expected

#### Observed Behavior
When rendering the card, the error ```Cannot read properties of undefined (reading 'value')``` occurs

#### Steps to Reproduce
Add an **Input.Text** control to a simple card
```
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.0",
  "body": [
    {
      "type": "Container",
      "items": [
        {
          "type": "Input.Text",
          "id": "pName",
          "label": "Name:",
          "placeholder": "enter a name"
        }
      ]
    }
  ],
  "actions": [
    {
      "type": "Action.Submit",
      "title": "Submit",
      "style": "positive",
      "associatedInputs": "auto",
      "data": {}
    }
  ]
}
```
Render the card using the ```AdaptiveCardHost``` control

The issue seems to lie in line 640 of the ```elements.tsx``` file when ```refControl``` is null or undefined.
https://github.com/araver/sp-dev-fx-controls-react/blob/7d033e51fff0fd5be2e5bd030a07a514ce7efe23/src/controls/adaptiveCardHost/fluentUI/Elements.tsx#L640

The following workaround resolves the issue (for me at least)

Define an override of the ```FluentUITextInput``` control
```
class FluentUITextInputFixed extends FluentUITextInput {
   public get value(): string | undefined {
      if (this.renderedInputControlElement) {
            return this?.["refControl"]?.value || undefined;
      } else {
            return undefined;
      }
   }
}
```
Unregister the existing ```FluentUITextInput``` control and register the new one
```
<AdaptiveCardHost
   ....
   onSetCustomElements={(registry: CardObjectRegistry<CardElement>) => {
         registry.unregister("Input.Text");
         registry.register("Input.Text", FluentUITextInputFixed);
   }}
/>
```

**Comment by patrikhellgren** [comment]
@phillipharding This has already been fixed in #1574 which will be part of the next release.

**Comment by phillipharding** [comment]
Thanks @patrikhellgren I'll close this issue
